content/protobuf/language-mapping/service.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Service
+description: Learn how Protobuf services are mapped to C#.
+---
+
+## Overview
+
+A Protobuf service specifies an abstraction—a group of abstract RPC methods. A client application calls these methods
+while a server application hosts a service that implements this service and all its methods.
```

Comment:
> a service that implements this service

This sounds odd to me.

Reply:
I agree, maybe we quality the second service "... a server application hosts a service which implements the Protobuf service".

Hmm, still sounds a bit awakward bit is a little bit cleaner.

Reply:
Maybe:

> while a server application hosts an implementation of this service and all its methods
?

Reply:
Used Austin's suggestion.